app/uk/gov/hmrc/uknwauthcheckerapi/errors/ApiErrorHandler.scala

```diff
@@ -37,12 +37,10 @@ class ApiErrorHandler @Inject() (appConfig: AppConfig) extends HttpErrorHandler
 
     Future.successful(
       statusCode match {
-        case FORBIDDEN          => ForbiddenApiError.toResult
-        case METHOD_NOT_ALLOWED => MethodNotAllowedApiError.toResult
+        case FORBIDDEN => ForbiddenApiError.toResult
         case NOT_FOUND =>
           request.path match {
```

Comment:
This doesn't need a match any more. You can just return NotFoundApiError.toResult